lib/datasource/docker/readme.md

```diff
@@ -0,0 +1,5 @@
+This datasource fetches the release note from a source repository specified according to the [pre-defined annotation keys of the OCI Image Format Specification](https://github.com/opencontainers/image-spec/blob/main/annotations.md).
```

Comment:
Datasources do not fetch release notes, so this is incorrect. I think you mean that it identifies the source URL of a package?

Reply:
```suggestion
This datasource allows to fetch the release note from a source repository specified according to the [pre-defined annotation keys of the OCI Image Format Specification](https://github.com/opencontainers/image-spec/blob/main/annotations.md).
```

@rarkins is it more accurate this way ? the next sentence explains how it works.